stock_location_donot_mix_order/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Stock Location Do Not Mix Order",
+    "summary": "Prevent to mix sales order on same consolidation location.",
```

Comment:
After rethinking on the name, I don't like that it starts with stock_location as it's like it's adding a feature on the location while it is a feature on a picking.
Maybe `stock_unique_order_per_location` ?
```suggestion
    "name": "Stock Unique Order Per Location",
    "summary": "Prevent to mix different sources on a same location.",
```

Reply:
I also renamed the related shopfloor module